Dockerfile.php5.6-busybox

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu:xenial as builder
+
+ENV PHP_VERSION 5.6.31
+ENV PECL_MODULES imagick-3.4.3 redis-3.1.3 memcached-2.2.0 ssh2-0.13 msgpack-0.5.7 igbinary-2.0.1 runkit-1.0.4
```

Comment:
This is where we define which pecl modules are installed and which versions of each to use. Everything related to the building and installation of the modules are driven by this ENV var.